pkg/arvo/sys/lull.hoon

```diff
@@ -1892,6 +1892,154 @@
+        $%  $>(%private-keys gift:jael)
+            $>(%public-keys gift:jael)
+            $>(%turf gift:jael)
+        ==  ==
+          $:  %khan
+        $%  $>(%arow gift:khan)
+        ==  ==
+      ==
+    +$  user-note
+      $~  [%b %rest *@da]
+      $%  $:  %a
+        $%  $>(%sift task:ames)
+            $>(%spew task:ames)
+            $>(%stir task:ames) :: helm does this
+            $>(%prod task:ames) :: helm does this
+            $>(%snub task:ames)
+        ==  ==
+          $:  %b
+        $%  $>(%rest task:behn)
+            $>(%wait task:behn)
+        ==  ==
+          $:  %c
+        $%  $>(%dirk task:clay)
+            $>(%mont task:clay)
+            $>(%ogre task:clay)
+            $>(%cred task:clay)
+            $>(%crew task:clay)
+            $>(%crow task:clay)
+            $>(%perm task:clay)
+            $>(%park task:clay)
+            $>(%info task:clay)
+            $>(%warp task:clay)
+            $>(%rein task:clay)
+            $>(%zest task:clay)
+            $>(%tire task:clay)
+            $>(%merg task:clay) :: kiln does this
+            $>(%fuse task:clay) :: kiln does this
+            $>(%drop task:clay) :: kiln does this
+            $>(%wick task:clay) :: kiln does this
+        ==  ==
+          $:  %d
+        $%  $>(%belt task:dill)
+            $>(%blew task:dill)
+            $>(%flee task:dill)
+            $>(%hail task:dill)
+            $>(%view task:dill)
+            $>(%crud task:dill)
+            $>(%talk task:dill)
+            $>(%text task:dill)
+            $>(%flog task:dill)
+            $>(%knob task:dill) :: helm does this
+            $>(%shot task:dill) :: herm does this
+        ==  ==
+          $:  %e
+        $%  $>(%connect task:eyre)
+            $>(%disconnect task:eyre)
+            $>(%rule task:eyre)
+            $>(%approve-origin task:eyre)
+            $>(%reject-origin task:eyre)
+            $>(%serve task:eyre)
+        ==  ==
+          $:  %g
+        $%  $>(%nuke task:gall)
+            $>(%deal task:gall)
+            $>(%spew task:gall) :: helm does this
+            $>(%sift task:gall) :: helm does this
+            $>(%doff task:gall) :: helm does this
+            $>(%sear task:gall) :: kiln does this
+        ==  ==
+          $:  %i
+        $%  $>(%request task:iris)
+            $>(%cancel-request task:iris)
+        ==  ==
+          $:  %j
+        $%  $>(%moon task:jael)
+            $>(%private-keys task:jael)
+            $>(%resend task:jael)
+            $>(%rekey task:jael)
+            $>(%step task:jael)
+            $>(%meet task:jael) :: docs say this task is deprecated and does not perform any actions
+            $>(%ruin task:jael)
+            $>(%nuke task:jael)
+            $>(%public-keys task:jael)
+            $>(%turf task:jael)
+            $>(%listen task:jael) :: base desk only
+        ==  ==
+          $:  %k
+        $%  $>(%fard task:khan)
+            $>(%lard task:khan) :: kiln does this
+        ==  ==
+      ==
```

Comment:
I'm very interested in comments of the list of userspace notes here. I created it by looking at Marks preliminary userspace permissioning work, then added all the moves that were needed for `%base` to compile. It is unlikely that I got it right the first time around.